client/src/components/relatedProducts/relatedCarousel.jsx

```diff
@@ -40,8 +40,11 @@ const responsive = {
 const RelatedCarousel = (props) => {
 
   return (
-    <div>
+    <div
+    id='relatedProducts'
+    data-testid = 'relatedCarousel'>
       <Carousel
```

Comment:
Related to the first comment, I think consistency is probably the only real standard to code by. The '>' is on a different line for these two.

Reply:
thank you